src/XeroResourceOwner.php

```diff
@@ -37,13 +37,39 @@ class XeroResourceOwner
      */
     public $family_name;
 
+    /**
```

Comment:
My thinking would be that these properties would stay inside a `jwt` property of this object, or they're assigned to more meaningful properties of the `XeroResourceOwner` 

Thoughts?

Reply:
Not sure. I see the JWT is a signed delivery method, and once it's decoded, the JWT itself has served its purpose; we are now dealing with data, or properties. The resource owner class could provide more meaning to the properties, such as providing timestamps as `DateTime` classes. The JWT custom claims have little awareness of what they are. I can see how accessing additional custom claims that could potentially come along in the future becomes a little more difficult.

Reply:
Just coming back to this, as I need to start using it in ernest.

Reply:
@judgej agreed. I think my original thinking was "why would a Xero Resource Owner have an expiry time" etc. As in, if you think about the object itself those properties don't make sense.  The problem here is probably because I created the `fromJWT()` method, which is really taking about a different thing.  

Re your comment about the JWT being discarded, if we're taking that approach, then should we be using the attribute names from it? It feels a little bit contradictory for that to be the goal, yet still retaining `sub`,`exp`,`aud` etc, which are meaningless outside a JWT context.

Reply:
Now I'm playing with it, I can see the expiry time makes sense here. The ID token is valid for only five minutes - it expires five minutes after it is issued. It's not used for access, so the expiry does not have any practical effect, but it just indicates that the identity should only be trust for five minutes. The names and email address could be changed at any time, so it is necessary to go back and fetch it again if you want to do anything serious with it.

Something else I have noticed, is that the identity token is also supplied with the new access token on a refresh. That is the only way of getting a fresh identity  token - refresh the access token and a new identity token comes along with it (so long as you are asking for it in the scopes).

Reply:
Just a heads up: a fix for the `updatedDateUtc` property of the tenant model is included in this PR. The API always returns a timestamp for that property, but the model always parses it to `null`.

Reply:
I think on the discarding comment, my mind was still in OAuth 1.0a mode. The full JWT needs to be kept, as that is the token that is sent with each API request. Some values it is worth pulling out separately - exp, xero_userid, scopes, but otherwise the rest of the data can stay in the JWT. They can be unwrapped and inspected if need be, but most of the time they won't be needed as individual properties.